sql/tasks.py

```diff
@@ -0,0 +1,123 @@
+from sqlalchemy import create_engine, extract
+from sqlalchemy.orm import sessionmaker
+from sql.entities import Task
+
+import config
+from constants import sql_engine, use_db_query
+from sql.helpers import entity_to_dict
+
+conf = config.DBConfig()
+
+
+#Todo(stasya) add logging to except blocks
+class SQLTasks:
+    def __init__(self):
+        self.engine = create_engine(sql_engine.format(
+            user=conf.user, password=conf.password, host=conf.host,
+            port=conf.port), pool_size=2)
+
+        with self.engine.begin() as conn:
+            conn.execute(use_db_query)
+
+        self.Session = sessionmaker(bind=self.engine)
+
+    def add_task(self, user_id, name, status=None, calendar_date=None):
+        session = self.Session()
+        task = Task(user_id, name, status, calendar_date)
+        try:
+            session.add(task)
+            session.commit()
+            query = session.query(Task).filter_by(id=task.id).first()
+            return entity_to_dict(query)

```

Comment:
There is more easy way -
https://stackoverflow.com/questions/1316952/sqlalchemy-flush-and-get-inserted-id
Please use - session.refresh()